scitos_docking/launch/charging.launch

```diff
@@ -1,13 +1,17 @@
 <launch>
   <arg name="machine" default="localhost" />
   <arg name="user" default="" />
-  <arg name="lightEBC" default="Port0_12V_Enabled" />
```

Comment:
Wasn't this a problem for some people who experienced their side PCs to be shut down? Sorry, I'm not quite sure.